layer-publisher/bin/layer-publisher.ts

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import { LayerPublisherStack } from '../src/layer-publisher-stack';
+
+const SSM_PARAM_LAYER_ARN = '/layers/powertools-layer-arn';
+
+const app = new cdk.App();
+new LayerPublisherStack(app, 'LayerPublisherStack', {
+  powerToolsPackageVersion: app.node.tryGetContext('PowerToolsPackageVersion'),
+  layerName: 'AWSLambdaPowertoolsNodeJS',
```

Comment:
Is there a possibility that two stacks are running at the same time? (Concerned about conflict with the same resource name)